src/modelfree/policies/wrappers.py

```diff
@@ -36,3 +36,44 @@ def predict(self, observation, state=None, mask=None, deterministic=False):
         noise = self.noise_generator(noise_param, action_shape)
         noisy_actions = original_actions * (1 + noise)
         return noisy_actions, states
+
+
+class MultiPolicyWrapper(DummyModel):
+    def __init__(self, policies, num_envs):
+        # TODO how do we do this properly, since DummyModel requires a single policy and sess?
+        super().__init__(policies, policies[0].sess)
+        self.policies = policies
+        # Can't currently easily read n_env off policies because they differ in n_env vs n_envs
+        self.num_envs = num_envs
+        self.action_space_shape = self.policies[0].policy.action_space.shape
+        for p in self.policies:
+            assert p.policy.action_space.shape == self.action_space_shape, "All policies must " \
```

Comment:
Line wrapping a bit awkward: think can do this more cleanly using parentheses?

Reply:
Flake also threw an error when I put it in parentheses, since it claims that the assertion is always true, which...doesn't seem to be the case to me?